tensorflow/core/common_runtime/process_util.cc

```diff
@@ -139,7 +160,7 @@ void SchedClosure(std::function<void()> closure) {
   uint64 id = tracing::GetUniqueArg();
   tracing::RecordEvent(tracing::EventCategory::kScheduleClosure, id);
 
-  Env::Default()->SchedClosure([id, closure = std::move(closure)]() {
+  Env::Default()->SchedClosure([ id, closure = std::move(closure) ]() {
```

Comment:
Just curious. Are the extra spaces the formatter's doing?

Reply:
yes.